test/fixtures/cookbooks/test/recipes/recursor_install_multi.rb

```diff
@@ -0,0 +1,31 @@
+pdns_recursor_install 'a_pdns_recursor' do
+  action :install
+  version version_per_platform
+end
+
+pdns_recursor_service 'a_pdns_recursor' do
+  action :enable
+end
+
+pdns_recursor_config 'a_pdns_recursor' do
+  action :create
+end
+
+pdns_recursor_install 'another_pdns_recursor' do
+  action :install
+  version version_per_platform
+end
+
+pdns_recursor_service 'another_pdns_recursor' do
+  action :enable
+end
+
+pdns_recursor_config 'another_pdns_recursor' do
+  action :create
+  run_user 'another-pdns'
+  run_group 'another-pdns'
+  run_user_home '/var/lib/another-pdns'
+  variables(
+    'local-port': '54'
```

Comment:
Is this a string or a symbol? Can be compacted to one line and if it's just a string, it should be `=>` and not re-casted to a symbol here.